smartcontractkit/external-adapters-js
# 18580 (PR 1) - URL Path Builder (method, unit tests, accuweather, amberdata, ap-election)

## Closes #18580

## Description
 Added buildUrlPath methods & unit tests to util. Updated the following adapters to use buildUrlPath: accuweather, amberdata, ap-election
......

## Steps to Test

1. yarn test:unit
2. yarn test:integration
3. Run an adapter with user input included in a url and check the URL it attempts to call.

## Quality Assurance
